README.md

```diff
@@ -203,6 +203,18 @@ version = "3.12"
     version = "3.18"
 ```
 
+Some records have non-standard `copy` or `free` functions or even do not have it.
+You can define these records with functions for boxed types.
```

Comment:
I think the name does not really reflect the intent here, a record might be defined as a boxed type but would not need to use g_boxed functions, how about `use_boxed_functions` maybe?

As for the documentation comment I would then go for:

```
gir auto-detects copy and free functions by looking up `type_name_copy` and `type_name_free` functions.
If the relevant functions are not named that way, and the type is defined with `G_DEFINE_BOXED_TYPE`,
`use_boxed_functions=true` can be used to call `g_boxed_copy` and `g_boxed_free` instead.

> In that case, it is required to declare the `get_type` function in the headers.
```

That is a bit verbose but would make me less confused :)

Reply:
Thanks, I make change